src/components/Avatar/Avatar.tsx

```diff
@@ -90,7 +100,7 @@ const Avatar: React.FC<AvatarProps> = props => {
         <Typography
           component='h4'
           variant={type === AvatarType.REGULAR ? 'subtitle1' : 'subtitle2'}>
-          {`${progress ? progress : 0} %`}
+          {progress && `${progress}%`}
```

Comment:
Shouldn't the progress be always defined? 
It should be always a value between 0 and 100? Worse case scenario is 0. 
In Figma we have a space between the progress and the "%"

Reply:
No because `Avatar` component is used in more than one place. In some views (check screenshot) we don't want to display "0" value 🤷‍♂️

:point_down:
![image](https://user-images.githubusercontent.com/14184100/68961761-627bdd00-07d3-11ea-9db5-4bfbc3149483.png)

Reply:
Why complicate in these 2 screens? If we take a closer look at these two screens what we might want is not our avatar component but only the material ui avatar component https://material-ui.com/components/avatars/ since we are just rendering a simple photo.

Other point is in all other screens that uses avatar photo we need to show progress even if it's 0%.
<img width="633" alt="Screenshot 2019-11-18 at 09 40 08" src="https://user-images.githubusercontent.com/12997200/69041471-76982800-09e7-11ea-9e79-29aebdd12838.png">

Reply:
>Other point is in all other screens that uses avatar photo we need to show progress even if it's 0%.

Oh right, forgot that `0` is evaluated as falsy value. Good catch :+1:. I fixed it d22ca83d502ccbd5af0c0c66d53587ab5f178b65
Now it's working when you pass `0` but progress won't be displayed, if `progress` is not passed via props. This way we can display avatar with or without props.

>Why complicate in these 2 screens? If we take a closer look at these two screens what we might want is not our avatar component but only the material ui avatar component https://material-ui.com/components/avatars/ since we are just rendering a simple photo, all other fields of our avatar component are not event used here (email, name, role, progress)

Hmmm... can't see nothing complicated here. I just re-use `Avatar` component. When photo is not present, placeholder will be displayed. Other than that there are some styles included. It's more convenient to re-use `Avatar` than using MaterialUI version 💪